src/plugin.ts

```diff
@@ -28,7 +28,7 @@ const plugin = {
 };
 
 export {
-	plugin as default,
+	CkeditorPlugin,
 	Ckeditor
```

Comment:
For me the name with uppercase "K" (`CKEditor` and `CKEditorPlugin`) would be more suitable. Especially since now we have named exports instead of default export. The Vue 3 guide in the docs also suggests naming the imported component as `CKEditor`: `import CKEditor from '@ckeditor/ckeditor5-vue';`.

Now, in addition to changing the default export to a named export, integrator will have to change the uppercase "K" to a lowercase "k": `import CKEditor from '@ckeditor/ckeditor5-vue';` ⟶ `import { Ckeditor } from '@ckeditor/ckeditor5-vue';`.

Reply:
See https://github.com/ckeditor/ckeditor5-vue/pull/290/commits/847e776784783499477a10f5b8f59f69e0bd7f5c